### PR TITLE
Feature: SKU Selector `displayPrices` prop

### PR DESCRIPTION
**Pull request by juliomoreira**
#### What problem is this solving?

-  Adds the option to show the SKU variation price in SKU Selector. This is best suited when `displayMode` is set to `select` but it works as well in `default` mode as shown bellow;

#### How to test it?

1. Open [Workspace](https://juliomoreira--drinkrunnr.myvtex.com/);
1. Scroll until the section "Summer";
1. Interact with the SKU Selector;
1. Observe that the item price is showing;

##### Product Summary

ezgif com-gif-maker (1)

##### Product Page

Screen Shot 2020-12-02 at 17 31 13

### CODE REVIEW

**Comment by juliomoreira** [comment]
Related to Issue: https://github.com/vtex-apps/store-components/issues/875

File: docs/SKUSelector.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ The SKU Selector is a product details page block responsible for displaying ever
...
 | `imageWidth` | `number` | `object` | Width (in `px`) of the product thumbnail image. You can declare an object as its value in case you want to define a width for each device (`desktop` and `mobile`).  | `undefined`  |
 | `initialSelection` | `enum`  | Controls the user initial selection for available variations when product page is fully loaded. Possible values are: `complete` (selects the first available SKU's variation values), `image` (selects the first available image variation) or `empty` (no variations will be selected when the page is loaded).  | `complete` |
+| `displayPrices`     | `boolean` | Only works if `displayMode` is set to `select` or `default`. Display the product price next to the variation label. Examples: [Shelf](https://user-images.githubusercontent.com/1207017/100927937-550bc080-34c4-11eb-9a2f-d0421f00e11b.png), [Product Page](https://user-images.githubusercontent.com/1207017/100927940-56d58400-34c4-11eb-911f-062c2f227511.png) | `false` |
```
Comment (guerreirobeatriz) [inline docs/SKUSelector.md:57]:
```suggestion
| `displayPrices`  | `boolean` | Whether the product price should be displayed next to the variation label (`true`) or not (`false`) - you can find practical examples of this prop working on a shelf and a product details page, respectively, [here](https://user-images.githubusercontent.com/1207017/100927937-550bc080-34c4-11eb-9a2f-d0421f00e11b.png) and [here](https://user-images.githubusercontent.com/1207017/100927940-56d58400-34c4-11eb-911f-062c2f227511.png). *Caution*: This prop only works if the `displayMode` prop is set as `select` or `default`.  | `false` |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

File: react/components/SKUSelector/components/SelectVariationMode.tsx [hunk 2]
```diff
@@ -19,12 +22,22 @@ const messages = defineMessages({
 })
 
 function SelectVariationMode(props: VariationSelectModeProps) {
-  const { intl, selectedItem, displayOptions } = props
+  const { intl, selectedItem, displayOptions, displayPrices } = props
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```
Comment (kaisermann) [inline react/components/SKUSelector/components/SelectVariationMode.tsx:27]: This is not needed now. Run `vtex setup --typings` and you should be good to go

File: react/components/SKUSelector/components/SelectVariationMode.tsx [hunk 3]
```diff
@@ -19,12 +22,22 @@ const messages = defineMessages({
...
+  const options = displayOptions.map(({ label, price: value }) => {
+    const price =
+      displayPrices && value && selectedItem !== label
+        ? ` - ${formatCurrency({ intl, culture, value })}`
+        : ''
+
+    return {
+      label: `${label}${price}`,
+      value: label,
+    }
```
Comment (kaisermann) [inline react/components/SKUSelector/components/SelectVariationMode.tsx:39]:
```suggestion
  if (displayPrices && value && selectedItem !== label) {
    return {
      label: `${label} - ${formatCurrency({ intl, culture, value })}`,
      value: label,
    }
  }

  return { label, value }
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

**Review by icazevedo — CHANGES_REQUESTED** [review]
The new prop should not be a boolean, but an enum. Maybe with `'current'` as a possible value, that would indicate that it will show the current prince, and being possibly undefined (default case).

**Pull request closed without merging**